### PR TITLE
Section 5 numbering update

### PR DESCRIPTION
**Pull request by jugonzal07**
Section 5 renumbering. 

- Old 5-16 removed.
- Old 5-17 now 5-16
- Old 5-18 now 5-17
- Old 5-29 now 5-18. 
- Old 5-30 and up are all one value lower (e.g., old 5-30 is new 5-29)

Also includes updates for changes in naming convention (e.g., RMR now RMD, RMD now RPD).

### CODE REVIEW

**Comment by weilixu** [comment]
I do not see the new 5-30 and 5-32 in the update list.
The new 5-38 (#1229), 5-39 (#1226) and 5-40(#1227) are still in the PR. Do you have chance to align the rule test jsons?

**Comment by weilixu** [comment]
@jugonzal07 all rule IDs are updated in this branch. But the CI is failing. Need to check the json tests.

**Comment by jugonzal07** [comment]
> I do not see the new 5-30 and 5-32 in the update list. The new 5-38 (#1229), 5-39 (#1226) and 5-40(#1227) are still in the PR. Do you have chance to align the rule test jsons?

Yup, I updated the rule test JSON numbers, the templates used to make them, and the rule IDs in the Rules tab.

5-30 (previously 5-31) and 5-32 (previously 5-33) did not appear in the latest envelope ruletest spreadsheet. Perhaps their TCDs have not been developed or are in another spreadsheet? The spreadsheet prior to updating the numbering can still be found archived here, timestamped as having last been updated 12/18/23.

https://1drv.ms/x/s!AoRQkE5gaWd5jTezCd1enfLrCS_K?e=jU6PJp

In other words, everything that existed in that 12/18/23 spreadsheet was updated to match the numbering updates we discussed earlier this week. If we're missing tests then either we missed developing them or perhaps @JacksonJ-KC has them offline?

**Comment by jugonzal07** [comment]
Ah-- actually, I think an issue might be that i did not remove a few JSONs that were deprecated with the update. I see what you're seeing @weilixu . For instance, 5-30 became 5-29 and since we dont have a 5-30 ruletest, the old one didn't get replaced.

I'll explore what's broken for a bit and get a final count on missing rule tests.

**Comment by jugonzal07** [comment]
All fixed. I removed deprecated rule tests. Rule tests 5-38 and 5-39 fail because those aren't in just yet, but the rest of the rule tests pass as expected. Nothing to update @JacksonJ-KC, sorry to ping you earlier.
